guides/doc-10-steps/Readme.md

```diff
@@ -0,0 +1,27 @@
+# 10 steps to an SOE
+Contains all contents needed to setup a document to make 10 steps.
```

Comment:
I do not have the context, but one idea that hits me is what if this grows up to 11 steps? Probably not the best title, maybe too not formal?